Jenkinsfile

```diff
@@ -19,17 +19,17 @@ spec:
     tty: true
     resources:
       requests:
-        ephemeral-storage: "20Gi"
+        ephemeral-storage: "30Gi"
```

Comment:
Is this neccesary as part of the PR?

Reply:
It was necessary to build the open telemetry on Jenkins, otherwise, the build would fail as it ran out of diskspace, Francesco upped it, I increased it again the next time we ran out of space, this is not a runtime feature, merely just enough to get through ci.